### PR TITLE
MCLOUD-6898: Add option to customize port for MailHog

### PR DESCRIPTION
**Pull request by BaDos**
## Purpose of this pull request

Added description of two new options for `ece-docker`

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-quick-reference.html
- https://devdocs.magento.com/cloud/docker/docker-config.html#set-up-email

## Links to Magento source code

https://github.com/magento/magento-cloud-docker/pull/290/files

whatsnew
Added MailHog configuration options to the `ece-docker build:compose` command to customize the HTTP and SMTP ports, and to disable MailHog. See [Set up email](https://devdocs.magento.com/cloud/docker/docker-config.html#set-up-email).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by meker12** [comment]
@BaDos It's a good idea to also update the [Set up email](https://devdocs.magento.com/cloud/docker/docker-config.html#set-up-email) topic with info about additional configuration options -- with a link or example. 
 
Does the next Docker release also make the installation of Mail Hog optional, or is it still installed by default?

**Review by meker12 — COMMENTED** [review]
Update Email set up topic with info about adding custom ports for Mail Hog.

File: src/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -107,6 +107,22 @@ The default {{ site.data.var.mcd-prod }} configuration includes the [MailHog] se
 
 When the MailHog service is installed, go to the following URL to open the service and view outgoing emails: `http://magento2.docker:8025`
 
+You can change HTTP and SMTP exposed ports using `--mailhog-http-port`<br>`--mailhog-smtp-port` options.
+
+For example:
```
Comment (meker12) [inline src/cloud/docker/docker-config.md:112]:
```suggestion
```

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

**Pull request merged**